lib/organisms/widgets-area/widgets-area.directive.js

```diff
@@ -21,12 +21,14 @@ function lnOWidgetsArea($compile) {
     elem.empty();
 
     angular.forEach(scope.lnWidgets, function(widget, index) {
-      var widgetDirective = '<section class="' + widget.type + '" ' + widget.type; 
+      var type = getWidgetType(widget.type);
+      var attrPrefix = (type === widget.type) ? '' : 'ln-';
```

Comment:
Yes, nowadays all the patternlab components have the "ln-" prefix for all their parameters. This line of code is just a way to make wp-widgets parameters (which come without "ln-" prefix) compatible with the patternlab standard.